index.js

```diff
@@ -42,89 +42,92 @@ function matchParent(el, selector) {
 const oldForm = document.querySelector('form');
 
 if (oldForm) {
-  const recipientTables = oldForm.querySelectorAll(':scope > table > tbody > tr > td > table');
-  const recipients = Array.from(recipientTables).map(table => {
-    const forwardToInput = table.querySelector('input[name^=senderforwardto]');
-    const email = decodeURIComponent(forwardToInput.name.slice('senderforwardto-'.length));
+  const moderationForm = oldForm.querySelector('input[name="submit"]');
+  if (moderationForm) {
```

Comment:
Ugh, the diff algorithm fails here. All I did was adding the two lines above and wrapping everything below in this ìf` block.